src/styles/_header.scss

```diff
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: 300ms;
+
+    &:hover {
+      color: #00addc;
```

Comment:
dont use `transition` for all, use only needed parametrs

Reply:
ok)

Reply:
i think, if we have only one change position, it can be used - "transition-duration" (as i made early)